.travis.yml

```diff
@@ -1,40 +1,40 @@
 sudo: required
-
 services:
-  - docker
-
+- docker
 env:
-  - DOCKER_COMPOSE_VERSION=1.11.2
-
+- DOCKER_COMPOSE_VERSION=1.11.2
 language: python
-
 python:
-  - "3.4"
-
+- '3.6'
 install:
-  - sudo apt-get update
-  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-engine
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
-  - sudo docker --version
-  - sudo docker-compose --version
-
+- sudo apt-get update
+- sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-engine
+- sudo rm /usr/local/bin/docker-compose
+- curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname
+  -s`-`uname -m` > docker-compose
+- chmod +x docker-compose
+- sudo mv docker-compose /usr/local/bin
+- sudo docker --version
+- sudo docker-compose --version
 before_script:
-  - ./csu start
-
+- "./csu start"
 script:
-  - ./csu dev test
-  - ./csu dev test_backwards
-  - ./csu dev test_coverage
-  - ./csu dev style
-  - ./csu dev docs
-
+- "./csu dev test"
+- "./csu dev test_backwards"
+- "./csu dev test_coverage"
+- "./csu dev style"
+- "./csu dev docs"
+cache:
+  directories:
+    - ${HOME}/google-cloud-sdk
```

Comment:
Possibly, from this documentation it references a file with sensitive info (but not where it is). Let's play it safe and not cache and rebuilt from scratch each time. 
https://cloud.google.com/storage/docs/gsutil/addlhelp/SecurityandPrivacyConsiderations

I had added it because sometimes the SDK was briefly unavailable for download.